recipe/meta.yaml

```diff
@@ -42,6 +42,9 @@ requirements:
     - cons
     - typing_extensions
     - setuptools >=48.0.0
+    # necessary to compile code at runtime
```

Comment:
Is it possible to configure without using a compiler at runtime https://pytensor.readthedocs.io/en/latest/tutorial/modes.html#default-modes?

Reply:
it probably is, but then `causalimpact` (or other similar packages that depend on `pytensor` and make use of its compiler feature) is breaking because it needs to compile at runtime.

`conda-forge` does this to `pytensor` too.
See https://github.com/conda-forge/pytensor-suite-feedstock/blob/57219359739dad904f860893c0eb30d00d71e934/recipe/meta.yaml#L90

There are also situations `pytensor` will start searching for headers like `<iostream>`, hence the need to put `compiler('cxx')` in `run`.

Reply:
I see. Thanks!